### DISCUSSION TITLE
clientSideAuth=0 parameter occasionally added to URL in the address bar and disables interaction with UI elements.

### DISCUSSION THREAD

**Discussion by q-timursirayev** [microsoft/Microsoft-Fabric-workload-development-sample#151] [Help]
Hi there
,
we have noticed that occasionally, some of us get **clientSideAuth=0** parameter attached to our URL when we do our testing. For example: https://app.fabric.microsoft.com/groups/me/list?experience=Org.xxxxxx.Product&clientSideAuth=0
I am not quite sure what exactly leads to it, but when it happens we can't create new items anymore from UI. Just nothing happens when you click on new item button. No errors in UI or Network too.
We can obviously work aroound it by removing the parameter from the URL in the browser but perhaps you could give us some clarification on the nature of this and how to avoid it in the first place.

This also happens quite often and then users are literally paralyzed which gives quite a negative image for our product.

![image](https://github.com/user-attachments/assets/e623640c-6dde-4f30-a462-73bf66b3a89a)

**Comment by ramizsha** [comment]
Hi @q-timursirayev,
We are aware of this problem and working on a fix.
Meanwhile the only mitigation is to delete the query parameter manually and refresh the page.
